src/sentry/tasks/weekly_escalating_forecast.py

```diff
@@ -36,16 +38,25 @@ def run_escalating_forecast() -> None:
     Run the escalating forecast algorithm on archived until escalating issues.
     """
     logger.info("Starting task for sentry.tasks.weekly_escalating_forecast.run_escalating_forecast")
+    for organization in RangeQuerySetWrapper(Organization.objects.all()):
+        for project_ids in chunked(
+            RangeQuerySetWrapper(
+                Project.objects.filter(
+                    status=ObjectStatus.ACTIVE, organization=organization
+                ).values_list("id", flat=True),
+                result_value_getter=lambda item: item,
+                step=ITERATOR_CHUNK,
+            ),
+            ITERATOR_CHUNK,
+        ):
+            if features.has("organizations:escalating-issues-v2", organization):
+                return generate_forecasts_for_projects.delay(
+                    project_ids=project_ids, forecast_for_ongoing=True
+                )
 
-    for project_ids in chunked(
-        RangeQuerySetWrapper(
-            Project.objects.filter(status=ObjectStatus.ACTIVE).values_list("id", flat=True),
-            result_value_getter=lambda item: item,
-            step=ITERATOR_CHUNK,
-        ),
-        ITERATOR_CHUNK,
-    ):
-        generate_forecasts_for_projects.delay(project_ids=project_ids)
+            return generate_forecasts_for_projects.delay(
```

Comment:
Should this be under an else block?

If not, you may be able to change `generate_forecasts_for_projects` to handle both cases in one shot, otherwise, you may be increasing the load twice.